### ISSUE TITLE
action is not compatible with docker/build-push-action

### ISSUE THREAD

**Issue by somdoron** [aevea/action-kaniko#9]
I think making them compatible will increase the usage of this action.

1. build-push-action has repository option vs image option on action-kanino.
2. dockerfile vs build_file
3. build_args is missing (comma separated)
4. tags is missing (comma separated)

@aexvir if you would like I can make a pull request to make them (more) compatible. However, it will probably break the current options.

**Comment by aexvir** [comment]
👋 Hi there @somdoron 
Hmm, not fully sure why making this action a drop-in replacement for docker's one is necessary, but I don't see any big reason against it either, so let's go for it.

I'd personally avoid making blatant breaking changes, but to be honest I'm not sure how much people is currently using the action... I don't think that adding a fallback to the current vars is too hard to do, and we can even display a deprecation notice for it. What do you think?

But yep, please feel free to submit a PR addressing this 🙂

**Comment by hadim** [comment]
I agree it would be a nice feature to have so it's possible to switch the "builder backend" by editing a simple line. The drawback is backward compatibility but since this action is young, it might actually be a good time to do it now before it's too late.

**Comment by somdoron** [comment]
I will give it a try with backward compatibility

**Comment by aexvir** [comment]
@somdoron did you end up giving a try to make this change?
Otherwise I'll look into it this weekend 🙂️
